salt/cloud/clouds/gce.py

```diff
@@ -78,6 +80,9 @@
         libcloud.security.CA_CERTS_PATH.append('/etc/ssl/certs/YaST-CA.pem')
     HAS_LIBCLOUD = True
 except ImportError:
+    LIBCLOUD_IMPORT_ERRORS.append("Failure importing libcloud!")
+    for line in traceback.format_exc().split("\n"):
+        LIBCLOUD_IMPORT_ERRORS.append(line)
```

Comment:
I suspect that instead of going through this we could change L163 to `log.exception` and just pass the raw exception to it.